code/game/atoms.dm

```diff
@@ -573,6 +592,82 @@
 
 	SEND_SIGNAL(src, COMSIG_PARENT_EXAMINE, user, .)
 
+/atom/proc/get_boozepower_text(booze_power, role)
+	if(isnull(booze_power))
+		return
+
+	if(role == "Bartender")
```

Comment:
This is disgusting, dont call this in the first place if they aren't a bartender and use a trait instead

Reply:
Also job strings use defines now

Reply:
`use a trait instead` what do you mean?

Reply:
look at examples of HAS_TRAIT

Reply:
I know, but which trait?

Reply:
Make one